.github/renovate.json5

```diff
@@ -98,6 +105,20 @@
           "third_party/**"
       ],
       "enabled": false
+    },
+    {
+      // Only patch level updates for container images in the imagevector. Minor and major versions are updated manually.
+      "matchDatasources": ["docker"],
+      "matchUpdateTypes": ["major", "minor"],
+      "matchFileNames": ["imagevector\/images\\.yaml"],
+      "enabled": false
+    },
```

Comment:
Why/what is the motivation about this? We could simply let the bot open PRs even for major/minor upgrades, no? This is also what happens today, at least for the images built in the `gardener` org.

Reply:
One motivation to update only patch versions is that minor/major versions usually require reading the release notes, additional adaptations to the component's helm chart (in our world the go code that deploys the container).

Reply:
Generally, I think the motivation for letting Renovate create PRs for new versions is to raise awareness that there is _something_ to update. Sometimes, additional changes are required, sometimes not (this does not only concern dependencies in the `images.yaml` but also others like `go.mod` dependencies). Such PRs can either be amended ([example](https://github.com/gardener/gardener/pull/9305/commits)) or closed and replaced by a dedicated PR, if desired. Still, people get aware. With patch versions only, we only tackle a part of the problem.

Reply:
The initial motivation for this was twofold:
1. This file already contains several similar cases.
2. I wanted to start conservative with the updates that should mostly work.

As @ialidzhikov mentioned, there are a number of dependencies, e.g. istio, where renovate will not do the right thing for major/minor updates by just updating the version. This configuration prevents those pull requests from even being opened by renovate. However, it indeed might leave us in the dark that there might be a new release.

I am open for both approaches.

Reply:
As mentioned previously in the Daily meeting, I prefer to include all releases (no matter whether major/minor/patch) for the reasons mentioned above. We can learn over time whether this is helpful or not, and adapt the configuration accordingly.
If we started with patch versions only, what would be the indicator that "now is a good time to also try major/minor"? I think it should rather be the other way round.
In case we already know that this won't work for some of the dependencies, we can explicitly exclude major/minor upgrades for them (like it is done for `k8s.io/*` or `sigs.k8s.io/controller-runtime` [here](https://github.com/gardener/gardener/blob/8f03dfcaaddafe8b58e39592c907852e8f2cffac/.github/renovate.json5#L65-L76)).

Reply:
Addressed with https://github.com/gardener/gardener/pull/9370/commits/a01d4433fd4607717ba673ec4f718c4e968cdaff and https://github.com/gardener/gardener/pull/9370/commits/96749bd179311a176d2b123d5a56e8ff55f1d307.

Reply:
>Generally, I think the motivation for letting Renovate create PRs for new versions is to raise awareness that there is something to update. Sometimes, additional changes are required, sometimes not (this does not only concern dependencies in the images.yaml but also others like go.mod dependencies). Such PRs can either be amended ([example](https://github.com/gardener/gardener/pull/9305/commits)) or closed and replaced by a dedicated PR, if desired. Still, people get aware. With patch versions only, we only tackle a part of the problem.

Sure, I agree. We also do the amend trick for PRs raised by dependabot for `github.com/gardener/gardener` dependency updates. We also have to be careful when reviewing/merging such PRs to check the upstream release notes and amend the required changes (if needed).